gtk4/src/param_spec_expression.rs

```diff
@@ -1,26 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+use crate::ParamSpecExpression;
+
 use glib::gobject_ffi;
 use glib::translate::*;
 use glib::ParamSpec;
 
-glib::wrapper! {
-    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
you forgot to remove it from lib.rs

Reply:
Also copy the value trait implementations from https://github.com/gtk-rs/gtk-rs-core/blob/master/glib/src/param_spec.rs#L37-L84  those were wrongly implemented in the past